client/galaxy/scripts/components/Toolshed/RepositoryDetails.vue

```diff
@@ -0,0 +1,177 @@
+            repoFields: {
+                numeric_revision: {
+                    label: "Revision"
+                },
+                tools: {
+                    label: "Tools and Versions"
+                },
+                profile: {
+                    label: "Requires"
+                },
+                missing_test_components: {
+                    label: "Tests"
+                },
+                actions: {
+                    label: ""
+                }
+            },
+            showSettings: false,
+            error: null,
+            loading: true,
+            delay: 2000
+        };
+    },
+    created() {
+        this.services = new Services();
+        this.loadDetails();
+    },
+    destroyed() {
+        this.clearTimeout();
+    },
+    methods: {
+        clearTimeout() {
+            if (this.timeout) {
+                clearTimeout(this.timeout);
+            }
+        },
+        loadDetails() {
+            this.services
+                .getDetails(this.toolshedUrl, this.repo.id)
+                .then(response => {
+                    this.repoTable = response;
+                    this.loadInstalledRepositories();
+                })
+                .catch(error => {
+                    this.error = error;
+                    this.loading = false;
+                });
+        },
+        listenInstalledRepositories() {
+            this.clearTimeout();
+            this.timeout = setTimeout(() => {
+                this.loadInstalledRepositories();
+            }, this.delay);
+        },
+        loadInstalledRepositories() {
+            this.services
+                .getInstalledRepositories(this.repo)
+                .then(revisions => {
+                    let changed = false;
+                    this.repoTable.forEach(x => {
+                        const revision = revisions[x.changeset_revision];
+                        if (revision && revision.status !== x.status) {
+                            x.status = revision.status;
+                            x.installed = revision.installed;
+                            changed = true;
+                            return false;
+                        }
+                    });
+                    if (changed) {
+                        this.repoTable = [...this.repoTable];
+                    }
+                    this.listenInstalledRepositories();
+                    this.loading = false;
+                })
+                .catch(error => {
+                    this.error = error;
+                    this.loading = false;
+                });
+        },
+        onOk: function() {
+            this.showSettings = false;
+        },
+        onHide: function() {
+            this.showSettings = false;
+        },
+        setupRepository: function(details) {
+            this.selectedChangeset = details.changeset_revision;
+            this.selectedRequiresPanel = details.includes_tools_for_display_in_tool_panel;
```

Comment:
How about
```suggestion
            this.selectedRequiresPanel = details.includes_tools_for_display_in_tool_panel || details.repository.type == "repository_suite_definition";
```
? That would display the panel selection when installing a suite.